test.ts

```diff
@@ -79,24 +151,98 @@ describe('supported value types', () => {
 
   });
 
+  // TODO: error "quiet": NaN "yesterday": NaN
   it('number with following hexadecimal value', () => {
-
+    const [options] = parse(
+      ['--yesterday=ff', '-q', 'ff'],
+      {
+        'yesterday': {
+          alias: 'y',
+          desc: 'oh I believe in yesterday.',
+          type: 'number'
+        },
+        'quiet': {
+          alias: 'q',
+          desc: 'the quiet one.',
+          type: 'number'
+        }
+      }
+    );
+    // expect(options).toEqual({
+    //   yesterday: 255,
+    //   quiet: 255
+    // });
   });
 
   it('number with following decimal value', () => {
-
+    const [options] = parse(
+      ['--yesterday=9.9', '-q', '9.9'],
+      {
+        'yesterday': {
+          alias: 'y',
+          desc: 'oh I believe in yesterday.',
+          type: 'number'
+        },
+        'quiet': {
+          alias: 'q',
+          desc: 'the quiet one.',
+          type: 'number'
+        }
+      }
+    );
+    expect(options).toEqual({
+      yesterday: 9.9,
+      quiet: 9.9
+    });
   });
 
+  // TODO: 设置进制？
   it('number with following octal value', () => {
-
```

Comment:
octal value is some value like '045'
octal is "number system and arithmetic based on the number **eight**"